libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -215,14 +324,120 @@ class AC_AutoTune {
         float current;
     } level_problem;
 
+    // parameters
     AP_Int8  axis_bitmask;
     AP_Float aggressiveness;
     AP_Float min_d;
+    AP_Float vel_hold_gain;
```

Comment:
It's odd that these parameters are declared in the parent class, AC_AutoTune,h but this class doesn't have a var_table.  We should probably discuss with @peterbarker but another option would be to leave AC_AutoTune with a var_table and then also allow the child classes to have var tables.  I'm sure we do this in other classes.

Reply:
I'll help with this.

Reply:
@rmackay9 moved code that changed gain values based on #defines to the subclasses.  I had used helper functions to get specific heli and multi defines but felt it better to put this code into the subclasses.  This moved all but one (axis_bitmask) parameter variable into the subclasses.  So I put those variable declarations into the subclass.  For axis_bitmask, I made an accessor so the parent class can access it and subsequently moved its declaration to the subclass.  Hope this makes sense and helps with your concern here.